scripts/python/docs-automation/sync_tocs.py

```diff
@@ -147,6 +182,14 @@ def get_non_standard_hrefs(doc_repo_location):
         default="",
         required=False,
     )
+    
+    parser.add_argument(
```

Comment:
What's the usage here?

Reply:
Got the answers. Thanks!

Reply:
Let's say I say `latest` as the moniker. That means that any hrefs with `<moniker>` will be filled in with that value. 

I'm currently planning on starting with `latest/` and `preview/` only, as we don't have any `legacy` stuff. It'll all work with that though!